subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/JvmVendorSpec.java

```diff
@@ -52,8 +52,10 @@ public abstract class JvmVendorSpec {
      * A constant for using <a href="https://developer.ibm.com/languages/java/semeru-runtimes/">IBM Semeru Runtimes</a> as the JVM vendor.
      *
      * @since 7.4
+     * @deprecated All IBM runtimes are Semeru runtimes, so no point in having two separate vendor definitions. IBM should be used instead.
```

Comment:
I do not think that first statement is correct.
It might be true for all IBM runtimes that are resolvable using the Foojay Dispo API.
But there are commercial IBM JDKs that are not Semeru runtimes, even if only old ones.

We should instead clarify that we are grouping all of them under the IBM vendor.